modules/kernel/infrastructure/db/sql/membership-eligibility-filter.ts

```
@@ -0,0 +1,63 @@
+import { and, eq, type SQL, sql } from "drizzle-orm";
+import {
+	FEATURE_ELIGIBLE_SUBSCRIPTION_STATUSES,
+	FEATURE_ELIGIBLE_USER_STATUS,
+} from "@/modules/kernel/domain/member-feature-eligibility";
+import { user } from "@/modules/kernel/infrastructure/db/schema";
```

Biome (2.4.9)

[error] All these imports are only used as types.

(lint/style/useImportType)